sdk/ml/azure-ai-ml/tests/dsl/unittests/test_dsl_pipeline_with_specific_nodes.py

```diff
@@ -1841,14 +1841,14 @@ def train_with_automl_in_pipeline(training_data, target_column_name_input: str):
 
     def test_pipeline_with_command_services(self):
         services = {
-            "my_ssh": {"job_service_type": "ssh"},
-            "my_tensorboard": {
-                "job_service_type": "tensor_board",
-                "properties": {
+            "my_ssh": JobService(job_service_type="ssh"),
```

Comment:
Does this run a full e2e test? is this good enough to say that pipeline with interactive job works?